support-files/mysql.server.sh

```diff
@@ -443,11 +443,8 @@ case "$mode" in
     exit $r
     ;;
   'bootstrap')
-      # Bootstrap the cluster, start the first node
-      # that initiate the cluster
-      echo $echo_n "Bootstrapping the cluster.. "
-      $0 start $other_args --wsrep-new-cluster
-      exit $?
+      log_failure_msg "Please use galera_new_cluster to start the mariadb service with --wsrep-new-cluster"
+      exit 1
```

Comment:
both /lib/lsb/init-functions and /etc/init.d/functions set _use_systemctl = 1 when systemd is available so I'll use that condition here.